core/src/main/java/io/questdb/cairo/DedupColumnCommitAddresses.java

```diff
@@ -101,12 +79,12 @@ public static long getColReserved5(long dedupBlockAddress, int keyIndex) {
         return Unsafe.getUnsafe().getLong(dedupBlockAddress + (long) keyIndex * RECORD_BYTES + RESERVED5);
     }
 
-    public static long getVarDataLen(long dedupBlockAddress, int keyIndex) {
-        return Unsafe.getUnsafe().getLong(dedupBlockAddress + (long) keyIndex * RECORD_BYTES + COL_VAR_DATA_LEN_64);
+    public static long getLagVarSize(long dedupBlockAddress, int keyIndex) {
+        return Unsafe.getUnsafe().getLong(dedupBlockAddress + (long) keyIndex * RECORD_BYTES + O3_VAR_DATA_LEN_64);
     }
 
-    public int getColumnCount() {
-        return columnCount;
+    public static long getVarDataLen(long dedupBlockAddress, int keyIndex) {
```

Comment:
nit: would be nice to call the methods consistently with each other and the setter methods, e.g. `getO3VarDataSize` and `getVarDataSize`

Reply:
Agree, renamed in [9e4f8bd](https://github.com/questdb/questdb/pull/4966/commits/9e4f8bd1838c52e3b2a62eb771b4f9b1b81738d2)